### ISSUE TITLE
EntityMetadataUtils.createPropertyPath doesn't manage Date object correctly

### ISSUE THREAD

**Issue by apegazblanc** [typeorm/typeorm#1037]
The property path of a Date column is not generated correctly.
In fact, that's return nothing instead of the path

In my case, the entity is not updated.
```
    const query:UpdateQueryBuilder<User> = this.connection
      .createQueryBuilder()
      .update(User)
      .set({disabled: new Date()})
      .where("id = :id", { id: existingUser.id });
    const res:any = await query.execute();
```

The property path of the field 'disabled' (a Date) is empty and the generated SQL is 
> UPDATE "schema"."users" SET  WHERE "id" = $2'

I think the issue is due to EntityMetadataUtils.createPropertyPath and the subPaths test, Date is matched as an Object.
```
if (entity[key] instanceof Object && !(entity[key] instanceof Function)) {
                const subPaths = this.createPropertyPath(entity[key], key);
                paths.push(...subPaths);
            }
```
that should be 
```
if (entity[key] instanceof Object && !(entity[key] instanceof Function) && !(entity[key] instanceof Date)) {
                const subPaths = this.createPropertyPath(entity[key], key);
                paths.push(...subPaths);
            }
```

The fix works locally for me.

**Comment by pleerock** [comment]
Thank you for the report. Fix for this issue will be released in `0.1.2`

**Comment by apegazblanc** [comment]
Thanks!
